packages/nimble-components/src/button/specs/README.md

```diff
@@ -0,0 +1,45 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary-mono` and `primary` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+[FAST Button API](https://github.com/microsoft/fast/blob/de7f234ef871204fcac2b5df59433d919809341d/packages/web-components/fast-foundation/src/button/README.md)
+
+The appearance of the button will have states `default` (White), `primary-mono` (Black), and `primary` (Green), each with its own color scheme based on the UI theme.
```

Comment:
I think trying to call out what color these states are is a bit confusing since it depends on what theme you are in. For example, `default` isn't "White" in most (or maybe any) cases and `primary-mono` isn't black in the Color theme.

This statement could also be clarified because there is something called `appearance` on a button, but you're really talking about `appearance-variant` here.

Perhaps something more like:
```suggestion
There are three possible values for `appearance-variant` on a button: `default`, `primary-mono`, and `primary`. Each variant will have its own color based on the UI theme.
```